src/types/timeline.ts

```diff
@@ -95,16 +98,26 @@ export type MouseDown = {
 };
 
 export type MouseOver = {
-  activityDirectives?: ActivityDirective[];
+  activityDirectivesByLayer?: Record<number, ActivityDirective[]>;
   constraintResults?: ConstraintResultWithName[];
   e: MouseEvent;
-  gaps?: Point[];
-  layerId: number;
-  origin?: MouseOverOrigin;
-  points?: Point[];
+  gapsByLayer?: Record<number, Point[]>;
+  layerId: number; //TODO not relevant since we sometimes have multiple layers per click
+  origin?: MouseOverOrigin; //TODO perhaps remove this
+  pointsByLayer?: Record<number, Point[]>;
   row?: Row;
   selectedActivityDirectiveId?: number;
   selectedSpanId?: number;
+  spansByLayer?: Record<number, Span[]>;
+};
+
+export type MouseOverEvent = Omit<
```

Comment:
Any way to better disambiguate `MouseOver` and `MouseOverEvent`?

Reply:
Renamed to `RowMouseOverEvent` to reflect usage and make it more consistent with `CellMouseOverEvent`